.github/dependabot.yml

```diff
@@ -4,3 +4,9 @@ updates:
     directory: /ecr-login
     schedule:
       interval: daily
+
+  # Dependencies listed in .github/workflows/*.yml
+  - package-ecosystem: "github-actions"
+    directory: "/.github/workflows"
```

Comment:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

> Set the directory to "/" to check for workflow files in .github/workflows.